samza-core/src/main/java/org/apache/samza/table/remote/couchbase/CouchbaseTableFunctionBase.java

```diff
@@ -38,6 +40,25 @@
   protected String bucketName;
   protected Cluster cluster;
   protected Bucket bucket;
+  protected Class<V> valueClass;
+  protected Serde<V> valueSerde;
+  protected long timeout = 0L;
+  protected TimeUnit timeUnit;
+  protected int ttl = 0; // default value 0 means no ttl, data will be storedforever
+
+  //TODO maybe we can create a builder class to create both read and write functions for the same bucket so that users don't need to type in the same things twice
+  public CouchbaseTableFunctionBase(Class<V> valueClass) {
+    this.valueClass = valueClass;
+  }
+
+  public void initial() {
```

Comment:
Why do we have this mehtod?

Reply:
It is used in the main function to test the methods. Since it's better to test it using end-to-end tests, we don't need it anymore.